docs/install.md

```diff
@@ -11,11 +11,13 @@
 
         sudo apt-get install redis-server python3.4 python3.4-dev python-pip python-virtualenv virtualenvwrapper
 
-1. create a virtualenv:
+2. create a virtualenv:
 
         mkvirtualenv addok --python=/usr/bin/python3.4
 
-1. install python packages:
+3. active virtualenv:
```

Comment:
It might be simpler that way. Otherwise it would be better to redirect **virtualenvwrapper** documentation for setup --> [virtualenvwrapper doc](http://virtualenvwrapper.readthedocs.io/en/latest/install.html#shell-startup-file)
Moreover **virtualenvwrapper** must be installed with pip on **OSX**.